File: ios/NativeClockSettingsModule.swift
import ExpoModulesCore

public class NativeClockSettingsModule: Module {
    public func definition() -> ModuleDefinition {
        Name("NativeClockSettings")

        Function("getElapsedTimeMillis") { () -> Double in
            let nanoseconds = clock_gettime_nsec_np(CLOCK_MONOTONIC_RAW)
            return Double(nanoseconds) / 1_000_000.0
        }
    }
}

File: android/src/main/java/expo/modules/nativeclocksettings/NativeClockSettingsModule.kt
package com.yourpackage.elapsedtime

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class NativeClockSettingsModule : Module() {
    override fun definition() = ModuleDefinition {
        Name("NativeClockSettings")

        Function("getElapsedTimeMillis") {
            android.os.SystemClock.elapsedRealtime()
        }
    }
}